css/styles.css

```diff
@@ -107,3 +107,80 @@ nav.navbar {
 	color: #f7f7f7;
 	font-size: 16px;
 }
+
+/* OUR STORY Section */
+
+#our_story,
+#review {
+	min-height: 600px;
+}
+
+#our_story .row {
+	margin-top: 30px;
+}
+
+#our_story h1 {
+	font-family: 'Lobster', sans-serif;
+	color: #f8512f;
+	font-size: 6em;
+	text-shadow: 1px 1px 5px #0a0a0a;
+}
+
+#our_story h3 {
+	font-family: 'Raleway', sans-serif;
+	font-weight: 700;
+	font-size: 2em;
+	margin-top: 0;
+}
+
+#our_story p {
+	font-family: 'Raleway', sans-serif;
+	font-weight: 300;
+}
+
+.story_year {
+	font-family: 'Playfair Display', sans-serif;
+	font-size: 200px;
+	font-weight: 700;
+	z-index: -1;
+	position: absolute;
+	color: rgba(149, 147, 147, 0.2);
+}
+
+@media only screen and (max-width: 768px) {
+	.story_year {
+		font-size: 150px;
+	}
+}
```

Comment:
this is no big issue now, but after PR has been merged. please move media Query at the bottom. looks great.